cl_sii/rtc/data_models_aec.py

```diff
@@ -648,6 +653,49 @@ def slug(self) -> str:
     # Custom Methods
     ###########################################################################
 
+    def verify_signature(
+        self,
+        xml_doc: xml_utils.XmlElement,
+    ) -> Optional[bool]:
+        """
+        Verify signature of AEC XML document ``xml_doc``.
+        :param xml_doc: An XML document, as returned by ``xml_utils.parse_untrusted_xml()``.
```

Comment:
There should be an empty line between the explanation text (`Verify sign...`) and `:param ...`.

```suggestion
        Verify signature of AEC XML document ``xml_doc``.

        :param xml_doc: An XML document, as returned by ``xml_utils.parse_untrusted_xml()``.
```